libs/design/src/core/openable/openable.directive.ts

```diff
@@ -58,20 +62,25 @@ import { DaffOpenable } from './openable';
   standalone: true,
 })
 
-export class DaffOpenableDirective implements DaffOpenable {
+export class DaffOpenableDirective implements DaffOpenable, OnChanges {
   /** Whether or not a component implementing the directive is open */
   @Input() @HostBinding('class.daff-open') open = false;
 
+  private _setOpen(v: boolean) {
+    if(!this.stateless) {
+      this.open = v;
```

Comment:
this should be mutating `this._open` not `this.open`. The setter for `this.open` contains the opposite check for `stateless` so this will never do anything.

Reply:
`_open` was removed, can you re-review?